libimage/manifests/manifests.go

```diff
@@ -73,6 +79,12 @@ type PushOptions struct {
 	SourceFilter                     LookupReferenceFunc   // filter the list source
 	AddCompression                   []string              // add existing instances with requested compression algorithms to manifest list
 	ForceCompressionFormat           bool                  // force push with requested compression ignoring the blobs which can be reused.
+	// Maximum number of retries with exponential backoff when facing
+	// transient network errors.  A reasonable default is used if not set.
+	// Default 3.
```

Comment:
No need to be vague by calling it "a reasonable default" if we're going to document that it's 3.